telco-core/configuration/core-baseline.yaml

```diff
@@ -77,13 +75,15 @@ policies:
     policyAnnotations:
       ran.openshift.io/ztp-deploy-wave: "7"
     manifests:
+      - path: reference-crs/optional/logging/ClusterLogServiceAccount.yaml
+      - path: reference-crs/optional/logging/ClusterLogServiceAccountAuditBinding.yaml
+      - path: reference-crs/optional/logging/ClusterLogServiceAccountInfrastructureBinding.yaml
+
+      # Moving to Cluster Logging 6.0 requires removing the 5.y API
       - path: reference-crs/optional/logging/ClusterLogging.yaml
-        patches:
-        - spec:
-            logStore:
-              type: lokistack
-              lokistack:
-                name: logging-loki
+        complianceType: mustnothave
+      - path: reference-crs/optional/logging/ClusterLogForwarder-removed.yaml
+        complianceType: mustnothave
```

Comment:
@imiller0 this one failed in my setup as referenced API wasn't found on the cluster

Reply:
Hmm. The "mustnothave" compliance type means that this CR should be removed by the policy (which is the intent b/c of the API change and this one isn't valid any longer).
Note also that I will be posting an update to this PR to move this to a different PolicyGenerator file to do this step in a later phase.

Reply:
> Hmm. The "mustnothave" compliance type means that this CR should be removed by the policy (which is the intent b/c of the API change and this one isn't valid any longer).

I was testing with fresh deployment so there was no API from _5.9_  but could be used for clusters that have it or during update

Reply:
Ah... So the mustnothave compliance can't deal with an undefined CR type... I'm going to split the upgrade piece into a separate PR as we sort that part out. Thanks!